src/Smalot/PdfParser/Page.php

```diff
@@ -658,6 +665,12 @@ public function getDataTm(array $dataCommands = null): array
          */
         $defaultTl = 0;
 
+        /*
+         *  Set default values for font data
+         */
+        $defaultFontId = -1;
+        $defaultFontSize = 0;
```

Comment:
Do these values matter later on? If yes, makes it sense to move them to `Config` instance too?

Reply:
It does not matter later on. I just used some values that could not be valid font id or font size.

Reply:
@k00ni btw, I wonder why the are separate cases for Tj and TJ here https://github.com/smalot/pdfparser/blob/master/src/Smalot/PdfParser/Page.php#L767,L814
Could this be joined? Now there are two cases that do the same thing